tests/test_association_task.py

```diff
@@ -258,20 +362,26 @@ def _store_dia_objects_and_sources(self):
                             0, 1, 2, 3, 4])
         for src_idx, dia_source in enumerate(dia_sources):
             dia_source['ccdVisitId'] = 1232
+            dia_source["midPointTai"] = \
+                dateTime.get(system=dafBase.DateTime.MJD)
+            dia_source["x"] = 0
+            dia_source["y"] = 0
             if src_idx >= n_objects:
                 dia_source['ccdVisitId'] += 1
             dia_source["psFlux"] = 10000 / self.flux0
-            dia_source["psFluxErr"] = np.sqrt(
-                (100 / self.flux0) ** 2 +
-                (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["psFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["apFlux"] = 10000 / self.flux0
+            dia_source["apFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["snr"] = 10
```

Comment:
Some code duplication with `_run_association_and_retrieve_objects` and `test_update_dia_objects`; can you combine how the three methods initialize sources?